service/rtc/session.go

```diff
@@ -417,9 +417,9 @@ func (s *session) addTrack(sdpOutCh chan<- Message, localTrack webrtc.TrackLocal
 		}
 	}
 
-	if getTrackType(localTrack.ID()) == trackTypeScreen && s.screenTrackSender != nil {
+	if getTrackType(localTrack.ID()) == trackTypeScreen && s.screenTrackSenders[getTrackMimeType(localTrack)] != nil {
```

Comment:
Still, no more than one track per encoding is expected (e.g. can't have 2 AV1 screen tracks).